docs/concepts/fundamentals/rendering_components.md

```diff
@@ -515,17 +518,18 @@ Button.render(
 )
 ```
 
-## Nesting components
```

Comment:
Renamed, because I realized that people may confuse this "nesting" with action of placing one `{% component %}` tag into another.